src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
```

Comment:
Error handling: Add error handling to prevent crashes when men.length is zero, which would cause division by zero.

For example=>

  if (men.length === 0) {
    throw new Error('There are no men on that list.');
  }